docs/src/main/asciidoc/security-openid-connect-client.adoc

```diff
@@ -329,58 +333,60 @@ org.acme.security.openid.connect.client.RestClientWithOidcClientFilter/mp-rest/u
 
-== Starting and Configuring the Keycloak Server
+== Starting and configuring the Keycloak server
 
-NOTE: Do not start the Keycloak server when you run the application in dev mode or test modes - `Dev Services for Keycloak` will launch a container. See <<keycloak-dev-mode,Running the Application in Dev mode>> section below for more information. Make sure to put the link:{quickstarts-tree-url}/security-openid-connect-client-quickstart/config/quarkus-realm.json[realm configuration file] on the classpath (`target/classes` directory) so that it gets imported automatically when running in dev mode - unless you have already built a link:{quickstarts-tree-url}/security-openid-connect-quickstart[complete solution] in which case this realm file will be added to the classpath during the build.
+NOTE: Do not start the Keycloak server when you run the application in dev mode or test modes; `Dev Services for Keycloak` launches a container. See the following <<keycloak-dev-mode,Running the Application in Dev mode>> section for more information. Make sure to put the link:{quickstarts-tree-url}/security-openid-connect-client-quickstart/config/quarkus-realm.json[realm configuration file] on the classpath (`target/classes` directory) so that it gets automatically imported when running in dev mode - unless you have already built a link:{quickstarts-tree-url}/security-openid-connect-quickstart[complete solution] in which case this realm file is added to the classpath during the build
 
-To start a Keycloak Server you can use Docker and just run the following command:
+NOTE: Do not start the Keycloak server when you run the application in dev or test modes; `Dev Services for Keycloak` launches a container. For more information, see the following <<keycloak-dev-mode,Running the Application in Dev mode>> section. Ensure you put the link:{quickstarts-tree-url}/security-openid-connect-client-quickstart/config/quarkus-realm.json[realm configuration file] on the classpath, in the `target/classes` directory. This placement ensures that the file gets automatically imported when running in dev mode. However, if you have already built a link:{quickstarts-tree-url}/security-openid-connect-quickstart[complete solution], you do not need to add realm file to the classpath because the build process has already done so.
+
+To start a Keycloak Server, you can use Docker and just run the following command:
 
 [source,bash,subs=attributes+]
 ----
 docker run --name keycloak -e KEYCLOAK_ADMIN=admin -e KEYCLOAK_ADMIN_PASSWORD=admin -p 8180:8080 quay.io/keycloak/keycloak:{keycloak.version} start-dev
 ----
 
-where `keycloak.version` should be set to `17.0.0` or higher.
+Where `keycloak.version` should be set to `17.0.0` or higher.
```

Comment:
```suggestion
Where `keycloak.version` should be set to `23.0.0` or higher.
```

Sorry, not part of this PR but the QE team noticed this is too old a version, and I've already started updating it in a couple of other docs, so please update here too

Reply:
Updated. Thanks.